articles/virtual-machines/workloads/sap/azure-monitor-sap-quickstart.md

```diff
@@ -78,13 +78,29 @@ Sign in to the Azure portal at https://portal.azure.com
 
 1. Select OS (Linux) from the drop-down 
 
-> [!IMPORTANT]
-> To configure OS (Linux) provider, ensure that Node_Exporter is installed in each BareMetal instance. For more information, see [Node_Exporter](https://github.com/prometheus/node_exporter)
+   > [!IMPORTANT]
+   > To configure OS (Linux) provider, ensure that Node_Exporter is installed in each BareMetal instance. For more information, see [Node_Exporter](https://github.com/prometheus/node_exporter).
+   >
+   > Be sure to download Node Exporter only from [Prometheus](https://prometheus.io/download/#node_exporter).
+   >
 
-2. Input a name, which will be the identifier for the BareMetal Instance.
+2. Input a name, which will be the identifier for the BareMetal instance.
 3. Input the Node Exporter Endpoint in the form of http://IP:9100/metrics.
 4. When finished, select **Add provider**. Continue to add more providers as needed or select **Review + create** to complete the deployment. 
 
+If Node Exporter can't connect with the OS provider on the BareMetal or VM instance, check the following settings:
+
+- Is Azure Monitor in the same VNet as the host machine?
```

Comment:
Can we be more explicit here?
Like - "Is your Azure Monitor resource deployed in the same VNet as the node (baremetal/vm) where node_exporter is deployed?"